src/System.Net.Security/src/System/Net/Unix/SslConnectionInfo.cs

```diff
@@ -54,16 +55,21 @@ private static CipherAlgorithmType MapCipherAlgorithmType(long encryption)
                     return CipherAlgorithmType.Rc4;
                 case 8:
                     return CipherAlgorithmType.Rc2;
-                case 16:
-                    return CipherAlgorithmType.None;
                 case 32:
                     return CipherAlgorithmType.Null;
                 case 64:
                     return CipherAlgorithmType.Aes128;
                 case 128:
                     return CipherAlgorithmType.Aes256;
+                case 16:
+                case 256:
+                case 512:
+                case 1024:
+                case 2048:
+                case 4096:
+                case 8192:
+                    return (CipherAlgorithmType)encryption;              
```

Comment:
Created the enums for the case label values.
As discussed ,local enums are not present in Security dll so cannot modify it . Going ahead with the constant values for unsupported enums